requirements/dev.txt

```diff
@@ -104,15 +95,16 @@ pytest-watch==4.2.0
 pytest-xdist==1.23.2
 pytest==3.9.2
 python-dateutil==2.7.4    # via faker, freezegun
+python-decouple==3.1
 python-oembed==0.2.4
 python-openid==2.2.5
 python-social-auth==0.3.6
+python3-openid==3.1.0     # via social-auth-core
 pytz==2018.6              # via django, django-paypal
 pyyaml==3.13              # via watchdog
 requests-oauthlib==1.0.0
-requests[security]==2.20.0
```

Comment:
why did it drop [security]?

Reply:
No clue - it's the `pip-compile` magic :)